src/runtime/database.cpp

```diff
@@ -396,6 +398,7 @@ std::string Database::open(bool wait, bool memory, bool tty) {
   const char *sql_tag_job = "insert into tags(job_id, uri, content) values(?, ?, ?)";
   const char *sql_get_tags = "select job_id, uri, content from tags where job_id=?";
   const char *sql_get_all_tags = "select job_id, uri, content from tags";
+  const char *sql_get_runs_table = "select run_id, time, cmdline from runs order by time ASC";
```

Comment:
Bikshedding nit: the `_table` suffix feels a bit out of place.  Yeah, you're effectively selecting the entire table, but the line just above this is doing something similar and is marked as such with an `_all_` infix instead.
```suggestion
  const char *sql_get_all_runs = "select run_id, time, cmdline from runs order by time ASC";
```